SignalRServiceBenchmarkPlugin/plugins/Plugin.Microsoft.Azure.SignalR.Benchmark/SignalRPlugin/MasterMethods/StopCollector.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Common;
+using Plugin.Base;
+using Rpc.Service;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Plugin.Microsoft.Azure.SignalR.Benchmark.MasterMethods
+{
+    public class StopCollector : IMasterMethod
+    {
+        public Task Do(IDictionary<string, object> stepParameters, IDictionary<string, object> pluginParameters, IList<IRpcClient> clients)
+        {
+            Log.Information($"Stop collecting...");
+
+            // Get parameters
+            stepParameters.TryGetTypedValue(SignalRConstants.Type, out string type, Convert.ToString);
+            pluginParameters.TryGetTypedValue($"{SignalRConstants.Timer}.{type}", out System.Timers.Timer timer, obj => (System.Timers.Timer) obj);
+
+            // Stop and dispose timer
+            timer.Stop();
```

Comment:
It is better use try-finally to handle exceptions.
try {
 ...
} finally {
  timer.Stop();
  timer.Dispose();
}

Reply:
Got it, thanks.